kogito-usertasks-custom-lifecycle-springboot/README.md

```diff
@@ -12,9 +12,33 @@ This example shows
 
 * working with user tasks
 * four eye principle with user tasks
-	
-	
+
+* Diagram	
 <p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+	
+Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties2.png"></p>
+
+* Diagram Properties
+<p align="center"><img src="docs/images/diagramProperties3.png"></p>
+
+* First Line Approval
+<p align="center"><img src="docs/images/firstLineApprovalUserTask.png"></p>
```

Comment:
please show only the properties panel to be in sync with other screenshots

Reply:
Hi @radtriste can you explain?

Reply:
see on Quarkus. can be let like that

Reply:
fixed @radtriste